.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Thanks for submitting a PR to the Genny repo. Please include the following fields (if relevant) prior to submitting your PR.
+
+**Jira Ticket covering work**
+
+**Whats Changed** (high level explanation of changes)
+
+**Patch testing results** (if applicable, link a patch test showing code changes running successfully)
+
+**Workload Submission form** (if applicable, only required if there is a new workload being added)
```

Comment:
woot woot! Can we add the link to the workload submission form in this template?